### PR TITLE
 added role to list pods

### PR DESCRIPTION
**Pull request by xrajesh**
Signed-off-by: Xavier Dharmaiyan <xavier@redhat.com>

**Related Issue:**  stolostron/backlog#<ISSUE_NUMBER>

### CODE REVIEW

**Review by jlpadilla — APPROVED** [review]
/lgtm

File: controllers/create_rolesbindings.go [hunk 1]
```diff
@@ -163,6 +163,11 @@ func getRules() []rbacv1.PolicyRule {
 			Resources: []string{"users", "serviceaccounts", "groups"},
 			Verbs:     []string{"impersonate"},
 		},
+		{
+			APIGroups: []string{""},
+			Resources: []string{"pods"},
+			Verbs:     []string{"list"},
```
Comment (jlpadilla) [inline controllers/create_rolesbindings.go:169]: Missing watch

**Comment by xrajesh** [comment]
Incorrect fix closing.

**Pull request closed without merging**